### ISSUE TITLE
Feature Request: Capability to fix the seed for all keras kernel initializers

### ISSUE THREAD

**Issue by mhaghighat** [tensorflow/tensorflow#31149]
**System information**
- TensorFlow version (you are using): 2.0-beta
- Are you willing to contribute it (Yes/No): No

**Describe the feature and the current behavior/state.**
If you create a Keras model, the only way to make your model reproducible is to repeat setting the seed for the `kernel_initializer` in every layer. This makes the code bulky and more prone to error. For example, a reproducible code looks like this:

```
np.random.seed(1)
tf.set_random_seed(1)
# FEATURE REQUEST for another parameter like the ones above that we can set
# here and avoid repeating initializer seed in each and every layer below

model = tf.keras.Sequential()
model.add(tf.keras.layers.Dense(8, activation='relu', kernel_initializer=keras.initializers.glorot_uniform(seed=1), input_shape=[1]))
model.add(tf.keras.layers.Dense(8, activation='relu', kernel_initializer=keras.initializers.glorot_uniform(seed=1)))
model.add(tf.keras.layers.Dense(1, activation='linear', kernel_initializer=keras.initializers.glorot_uniform(seed=1)))
```
I wonder if you can add a default seed parameter that we can set and it affects all the initializers. 

**Will this change the current api? How?**
No

**Who will benefit with this feature?**
Anyone who likes clean code and would like to create reproducible models.

**Comment by jvishnuvardhan** [comment]
@mhaghighat Sorry for the late response. 
Look like you are looking for the following feature.
https://www.tensorflow.org/api_docs/python/tf/config/experimental/enable_op_determinism

If you think the above is not the feature you are looking, please feel free to open a PR in  [keras-team/keras repo.](https://github.com/keras-team/keras/issues) repository.

Please note that Keras development moved to keras-team/keras repository to focus on only keras. Thanks!
